internal/httpproxy/http_proxy.go

```diff
@@ -386,31 +386,37 @@ func (p *Proxy) Start(ready chan struct{}) {
 }
 
 func (p *Proxy) serveHTTP(w http.ResponseWriter, r *http.Request) {
-	logger := zap.S()
+	logger := zap.L().With(
```

Comment:
Use `Logger` instead of `SugarLogger` mostly because I find it easier to read to use `zap.String()` to keep key and value pair together, and `Logger` only allows this syntax.